packages/cli-platform-android/src/config/index.ts

```diff
@@ -132,10 +132,12 @@ export function dependencyConfig(
   const androidMkPath = userConfig.androidMkPath
     ? path.join(sourceDir, userConfig.androidMkPath)
     : path.join(sourceDir, 'build/generated/source/codegen/jni/Android.mk');
-  const cmakeListsPath = userConfig.cmakeListsPath
+  let cmakeListsPath = userConfig.cmakeListsPath
     ? path.join(sourceDir, userConfig.cmakeListsPath)
     : path.join(sourceDir, 'build/generated/source/codegen/jni/CMakeLists.txt');
-
+  if(process.platform == "win32") {
+    cmakeListsPath = cmakeListsPath.replaceAll("\\", "/")
```

Comment:
According to this table, `replaceAll` is only available from Node 15+: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll#browser_compatibility

Should we use `replace` with regex instead?

```suggestion
    cmakeListsPath = cmakeListsPath.replace(/\\/g, "/");
```